src/Controller/UserController.php

```diff
@@ -80,50 +81,55 @@ public function getUserDetails(User $user): Response
      * @throws ORMException
      * @throws OptimisticLockException
      */
-    public function editUser($id, Request $request, UserRepository $userRepository)
+    public function editUser($id, Request $request, UserRepository $userRepository, UserInterface $user)
```

Comment:
I would use `$currentUser` or `$loggedUser` or `$authenticatedUser` as it better reflects what kind of user is this.